src/Appwrite/Utopia/Response/Model/UsageProject.php

```diff
@@ -88,6 +94,13 @@ public function __construct()
                 'example' => [],
                 'array' => true
             ])
+            ->addRule('deploymentsStorageBreakdown', [
```

Comment:
```suggestion
            ->addRule('deploymentsBreakdown', [
```
I think this should suffice